src/Microsoft.Sbom.Api/README.md

```diff
@@ -1,9 +1,27 @@
-﻿Generates Software Bill of Materials (SBOM)
+Generates Software Bill of Materials (SBOM)
+
+See [SBOM API Reference](https://github.com/microsoft/sbom-tool/blob/main/docs/sbom-tool-api-reference.md) for details on how to set up a Generation Service using the Microsoft Dependency
+Injection Library.
 
 #### Scan Sample
 ```C#
-            var generator = new SBOMGenerator();
+using Microsoft.Extensions.Hosting;
+using Microsoft.Sbom.Contracts;
+
+namespace SBOMApiExample
+{
+    public class GenerationService: IHostedService
```

Comment:
Why change this from a generic example to an ASP.NET specific example?

Reply:
The old example showed the generator being instantiated without arguments. This was possible when we were using ninject in v0.3.3. Now that we are using Microsoft Dependency injection libraries these arguments need to be passed into the SBOMGenerator object. It is technically possible for users to do this manually but would be much more complex than this approach.

Reply:
My issue wasn't about the arguments, but `IHostedService` is ASP.NET. Generating SBOMs in ASP.NET is possible, but I wouldn't see it as a common scenario.